shell/src/main/java/alluxio/cli/fsadmin/command/DeleteWorker.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command;
+
+import static alluxio.Constants.LOST_WORKER_STATE;
+
+import alluxio.client.block.options.GetWorkerReportOptions;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.AlluxioException;
+import alluxio.client.block.options.GetWorkerReportOptions.WorkerInfoField;
+import alluxio.wire.WorkerInfo;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * todo.
+ */
+public class DeleteWorker extends AbstractFsAdminCommand {
+  private static final Option HELP_OPTION =
+      Option.builder()
+          .required(false)
+          .hasArg(false)
+          .longOpt("help")
+          .desc("print help information.")
+          .build();
+  private static final Option FORCE_FLAG =
+      Option.builder()
+          .required(false)
+          .hasArg(false)
+          .longOpt("force")
+          .desc("print help information.")
+          .build();
+
+  @Override
```

Comment:
All overloaded methods should be placed next to each other. Placing non-overloaded methods in between overloaded methods with the same type is a violation